acl.js

```diff
@@ -284,11 +286,46 @@ function allowOrigin(mode, userId, res, aclGraph, subject) {
     }
 }
 
+function fetchDocument(graph, uri, req) {
+    var options = req.app.locals.ldp;
+    var uriBase = file.uriBase(req);
+
+    var body = "";
+    if (S(uri).startsWith(uriBase)) {
+        try {
+            var oldPath = req.url;
+            var newPath = S(uri).chompLeft(uriBase).s;
+            var documentPath = file.uriToFilename(newPath, options.root);
```

Comment:
Sorry, I just re-read the code. I just saw the `S(uri)`, my bad.

I was pointing out the following, but I realized now, it is not inherent!

```
original URL == 'https://localhost:2000/myMountPath/path/to/abc.ttl'
req.originUrl == '/myMountPath/path/to/abc.ttl'
req.path == '/path/to/abc.ttl'
file.uriBase(req) == 'https://localhost:2000/myMountPath'
```

r+